freeorion.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>freeorion.desktop</id>
```

Comment:
Dubious `id`, according to https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-filespec it should be a reverse domain notation e.g. `org.freeorion`.

Probably need to cross check with `freeorion.desktop` file.

Reply:
Fixed bf4bdf49ae5354c415621a9657ae63c0602bb493

Reply:
You can't just change this without renaming `freeorion.desktop` as well.

Reply:
@Mailaender thanks for pointing out, fixed in b52d6bc303eac68ef1c69d4adf976b55603c4602